go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/ossf/allstar
 
-go 1.23.0
+go 1.24
```

Comment:
`1.24.0`?

`1.24` assumes a `.0`and works with current tooling but I remember the initial tooling in Go 1.21 or 1.22 throwing errors at this format.